packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -1,17 +1,224 @@
+    { key: 'artist', label: 'Artist' },
+    { key: 'album', label: 'Album' },
+    { key: 'year', label: 'Year' },
+  ]);
+  context.set('sortBy', 'artist');
+  context.set('sortOrder', 'asc');
+};
+const renderSortableTable = async () => {
+  await render(hbs`
+  <Hds::Table
+        @model={{this.model}}
+        @sortBy={{this.sortBy}}
+        @sortOrder={{this.sortOrder}}
+        @columns={{this.columns}}
+        id="data-test-table"
+      >
+        <:body as |row|>
+          <Hds::Table::Tr>
+            <td>{{row.artist}}</td>
+            <td>{{row.album}}</td>
+            <td>{{row.year}}</td>
+          </Hds::Table::Tr>
+        </:body>
+      </Hds::Table>
+  `);
+};
+
 module('Integration | Component | hds/table/index', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders the component', async function (assert) {
     await render(hbs`<Hds::Table />`);
     assert.dom(this.element).exists();
   });
+
   test('it should render with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`<Hds::Table id="data-test-table"/>`);
+    assert.dom('#data-test-table').hasClass('hds-table');
+  });
+
+  test('it should render with a CSS class appropriate for the @density value', async function (assert) {
+    await render(hbs`<Hds::Table @density="short" />`);
+    assert.dom('[data-test-table]').hasClass('hds-table--density-short');
+  });
+
+  test('it should render with a CSS class appropriate if no @density value is set', async function (assert) {
     await render(hbs`<Hds::Table />`);
-    assert.dom('[data-test-table]').hasClass('hds-table');
+    assert.dom('[data-test-table]').hasClass('hds-table--density-medium');
+  });
+
+  test('it should support splattributes', async function (assert) {
+    await render(
+      hbs`<Hds::Table id="data-test-table" aria-label="data test table" />`
+    );
+    assert
+      .dom('#data-test-table')
+      .hasAttribute('aria-label', 'data test table');
+  });
+
+  test('it should render the table with manual data passed and no model defined', async function (assert) {
+    await render(hbs`
+      <Hds::Table id="data-test-table">
+        <:head>
+          <Hds::Table::Tr>
+            <Hds::Table::Th>Cell Header 1</Hds::Table::Th>
+            <Hds::Table::Th>Cell Header 2</Hds::Table::Th>
+            <Hds::Table::Th>Cell Header 3</Hds::Table::Th>
+          </Hds::Table::Tr>
+        </:head>
+        <:body>
+          <Hds::Table::Tr>
+            <td>Cell Content 1 1</td>
+            <td>Cell Content 1 2</td>
+            <td>Cell Content 1 3</td>
+          </Hds::Table::Tr>
+          <Hds::Table::Tr>
+            <td>Cell Content 2 1</td>
+            <td>Cell Content 2 2</td>
+            <td>Cell Content 2 3</td>
+          </Hds::Table::Tr>
+          <Hds::Table::Tr>
+            <td>Cell Content 3 1</td>
+            <td>Cell Content 3 2</td>
+            <td>Cell Content 3 3</td>
+          </Hds::Table::Tr>
```

Comment:
since we're testing only the first row, we can remove the other two and reduce the overall code

Reply:
I'm confused by this comment. Does this have an effect on how the code functions?